tests/test-sources/plugins/pluginmanagers/packer.nix

```diff
@@ -136,10 +137,19 @@
     };
   };
 
-  no-packages = {
+  no-packages = cfg: {
     plugins.packer = {
       enable = true;
       gitPackage = null;
     };
+
+    assertions = [
+      {
+        assertion =
+          cfg.config.extraPackages != [ ]
+          -> lib.any (x: x.pname != null && x.pname != "git") cfg.config.extraPackages;
```

Comment:
If `x` is not an attrset that contains `pname`, then this will throw an eval error. You probably want `x.pname or null != "git"`